bitnami/contour/values.yaml

```
@@ -112,8 +112,14 @@ contour:
     ##
     pullSecrets: []
     debug: false
-  ## @param contour.replicaCount Number of Contour Pod replicas
-  ##
+  ## @param contour.contourConfigName Contour Deployment with Contourconfiguration CRD.
```

This was not updated properly. According to the documentation, they refer to it as ContourConfiguration

https://projectcontour.io/docs/main/configuration/

```suggestion
  ## @param contour.contourConfigName Contour Deployment with ContourConfiguration CRD.
```